docs/sql-guide/functions/function-vector-distances.md

```diff
@@ -1,24 +1,35 @@
 ---
-title: EUCLIDEAN_DISTANCE()
+title: vector distance functions
 layout: default
 parent: Functions
 grand_parent: SQL guide
 ---
-# EUCLIDEAN_DISTANCE() function
+# Vector distance functions
 
-`EUCLIDEAN_DISTANCE()` is a mathematical function that can be performed on `VECTOR` data type columns.
-The Euclidean (or Euclidian) distance between two vectors is the square root of the sum of the squares
-the pairwise differences between the vectors. This is the distance metric most commonly used when talking
-about distances between locations on a plane.
+Two functions are available to measure the difference between two vectors. They have
```

Comment:
How about:

COSINE_DISTANCE() or EUCLIDEAN_DISTANCE() measure the difference between two vectors.

I say this because the syntax is immediately below, so users can see it's almost identical, and the args explain what the differences are :D